app/models/service_template_ansible_playbook.rb

```diff
@@ -104,7 +104,7 @@ def self.build_parameter_list(name, description, info)
       :ask_limit_on_launch      => true,
       :ask_inventory_on_launch  => true,
       :ask_credential_on_launch => true
-    }
+    }.merge(info.slice(:become_enabled))
```

Comment:
Why not set this with the other parameters?
`:become_enable => info[:become_enabled]`
or if the API cannot support the property possibility being `nil`
`:become_enable => info[:become_enabled] || false`